### DISCUSSION TITLE
Unpausing should be instantaneous

### DISCUSSION THREAD

**Discussion by hburn7** [ppy/osu#20768] [Ideas]
Currently in Lazer, there is an audible and visual ramp up that happens after unpausing which feels terrible when trying to unpause during higher intensity moments. The audio played when resuming is distracting. On stable there is zero ramp up or delay when clicking the orange circle to resume gameplay. I hope this could at least be implemented as a setting. It feels really bad to miss after unpausing due to this.

**Comment by peppy** [comment]
This was half-intentional to avoid users over-using pausing during gameplay. Still open to discussion, but I don't htink users should be overly pausing (outside of break time) if possible.

**Reply by hburn7** [reply]
I can definitely see why you think this is an issue. However I think this should be reconsidered, perhaps limiting the number of times you can pause during a map would be a better alternative if your goal is to prevent abuse? I feel like this punishes pausing even once which I often do for legitimate reasons (e.g. I really need to scratch my nose right in that moment).

**Reply by peppy** [reply]
The ramp time is only 200ms, so it should not affect you if you paused outside of active segments.

**Reply by hburn7** [reply]
More often than not the maps I play never have pauses in intensity or break time, or the breaks are minutes apart from each other. This is typical on maps over 6 stars which is all I play at this point. Over the years I've learned how to pause and resume in the middle of a slightly slower slider to compensate for this when I need to pause for a few seconds and take care of something like an itchy nose. The ramp time can be learned in a similar way. It feels like players who are pausing and unpausing for legitimate reasons are being punished. As an aside I am also of the opinion that players who pause-abuse do so during break time anyway to regain stamina - unpausing during gameplay requires a lot of skill, especially when continuing a high accuracy or full-combo play.

**Reply by peppy** [reply]
I don't want to spur another pause discussion, just stating the reason it exists for now. We'll do another pass of this area in the future.
